stellargraph/datasets/datasets.py

```diff
@@ -707,3 +707,42 @@ def load(self):
             ``graph``.
         """
         return _load_tsv_knowledge_graph(self)
+
+
+class IAEnronEmployees(
+    DatasetLoader,
+    name="IAEnronEmployees",
```

Comment:
The website styles this dataset as 

```suggestion
    name="ia-enron-employees",
```

but I don't know if that's the canonical name by which people usually refer to it.